### PR TITLE
First sketch for scan request function

### PR DESCRIPTION
**Pull request by manishsat**
#### Description of changes

First sketch for scan request function
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/web-api/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "name": "web-api",
+    "version": "1.0.0",
+    "description": "This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.",
+    "scripts": {
+        "build": "webpack --config ./webpack.config.js --config-name runner --colors --progress",
```
Comment (devaradhanm) [inline packages/web-api/package.json]: change config name

File: packages/web-api/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const copyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = env => {
+    const version = env ? env.version : 'dev';
+    console.log(`Building for version : ${version}`);
+
+    return {
+        devtool: 'cheap-source-map',
+        externals: ['applicationinsights'],
```
Comment (devaradhanm) [inline packages/web-api/webpack.config.js]: we may not need to bundle any external package, if you are using container

File: packages/web-api/scan-request/function.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "bindings": [
+        {
+            "name": "scanRequest",
+            "type": "httpTrigger",
+            "direction": "in"
+        },
+        {
+            "type": "http",
+            "direction": "out",
+            "name": "res"
```
Comment (lamaks) [inline packages/web-api/scan-request/function.json]: can it be more meaningful?

Comment (manishsat) [reply packages/web-api/scan-request/function.json]: You mean name?

File: packages/web-api/package.json (hunk 1 above)
Comment (manishsat) [reply packages/web-api/package.json]: CP mistake, will change

File: packages/web-api/scan-request/function.json (hunk 3 above)
Comment (lamaks) [reply packages/web-api/scan-request/function.json]: yes name == res is this pre-set or custom?

Comment (manishsat) [reply packages/web-api/scan-request/function.json]: we can have any name here, I will change to name: scanResponse, this name will be available on context to assign response content.

**Pull request merged**